.bazelignore

```diff
@@ -6,3 +6,5 @@ bazel-rules_python
 bazel-bin
 bazel-out
 bazel-testlogs
+examples/bzlmod/bazel-bzlmod
```

Comment:
@rickeylev any reason these are here?

Reply:
Yes, they make `bazel test ...` (from the rules_python workspace) ignore the generated symlinks in the example sub-directory workspaces. Otherwise it'll recurse into them and find the various BUILD files and try to interpret them.